packages/expo-application/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unpublished
 
+- Updated `robolectric` to `4.10` and `junit` to `4.13.2`.  ([#22395](https://github.com/expo/expo/pull/22395) by [@josephyanks](https://github.com/josephyanks))
+
```

Comment:
```suggestion

### 📚 3rd party library updates

- Updated `robolectric` to `4.10` and `junit` to `4.13.2`.  ([#22395](https://github.com/expo/expo/pull/22395) by [@josephyanks](https://github.com/josephyanks))

```

thanks for the great pr! most changes look good to me. it's just one nit for changelog. would you mind to update all changelog under a section like `3rd party library updates`?

Reply:
Updated them all, thanks